### PR TITLE
Feat newsfeed

### PR DESCRIPTION
**Pull request by Torfeltorfel**
Add newsfeed on home screen & dynamic unsplash images :)

### CODE REVIEW

**Comment by Torfeltorfel** [comment]
Home screen ready 🎉

File: home.html [hunk 1]
```diff
@@ -19,7 +19,43 @@ <h1>Good Morning, Alex.</h1>
     </header>
 
     <main class="main">
-      <section>Newsfeed</section>
+      <section class="newsfeed">
+          <ul class="newsfeed__list">
+            <li>
+              <div class="content-container">
```
Comment (badabam) [inline home.html:25]: No need for this `div`. Just put those classes on the `li`. Also: please consider  a different class name as this is very generic (but long).

File: home.html [hunk 2]
```diff
@@ -19,7 +19,43 @@ <h1>Good Morning, Alex.</h1>
...
+                <div class="content-container__wrapper">
+                  <img class="content-container__wrapper--avatar" src="https://source.unsplash.com/100x100/?men" alt="newsfeed user">
+                </div>
+              </div>
+            </li>
+            <li>
+              <div class="content-container">
+                <img class="content-container__image" src="https://source.unsplash.com/200x280?airplane" alt="newsfeed image">
+                <div class="content-container__wrapper">
+                  <img class="content-container__wrapper--avatar" src="https://source.unsplash.com/100x100/?women" alt="newsfeed user">
```
Comment (badabam) [inline home.html:36]:
```suggestion
                  <img class="content-container__avatar" src="https://source.unsplash.com/100x100/?women" alt="newsfeed user">
```

File: postView.html [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (badabam) [inline postView.html:7]: Please delete this stump of a file. 😄

File: styles/components/newsfeed.css [hunk 4]
```diff
@@ -0,0 +1,46 @@
+.content-container {
+    height: 140px;
```
Comment (badabam) [inline styles/components/newsfeed.css:2]: Please find a way to avoid the fixed width. What if someone has a bigger font size?

**Review by badabam — COMMENTED** [review]
A lot of great stuff. Found some things to improve for you to consider.

**Pull request merged**